payment-method-messaging/src/test/java/com/stripe/android/paymentmethodmessaging/PaymentMethodMessagingViewModelTest.kt

```diff
@@ -0,0 +1,95 @@
+package com.stripe.android.paymentmethodmessaging
+
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.model.PaymentMethodMessage
+import com.stripe.android.networking.StripeApiRepository
+import com.stripe.android.paymentmethodmessaging.view.PaymentMethodMessagingView
+import com.stripe.android.paymentmethodmessaging.view.PaymentMethodMessagingViewModel
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+import org.mockito.kotlin.anyOrNull
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class PaymentMethodMessagingViewModelTest {
+    private val stripeApiRepository = mock<StripeApiRepository>()
```

Comment:
What do you think of adding a `FakeStripeRepository` instead of using a mock here?

Reply:
Since this is in a different module, I don't have access to `AbsFakeStripeRepository`. Should I move `AbsFakeStripeRepository`?